### ISSUE TITLE
Better Flexbox Support

### ISSUE THREAD

**Issue by eljefe6a** [astefanutti/decktape#44]
My Remark slides use Flexbox extensively. After many hours I finally figured out how to make my slides kind of work with Phantom 2.0's Flexbox support. Are there plans to update to Phantom 2.1? It has much better Flexbox support.

**Comment by astefanutti** [comment]
@eljefe6a the improved PhantomJS version maintained for DeckTape (astefanutti/phantomjs@cc06f444e4778d181e4df9523077177a6dcb4264) is already rebased few commits ahead of PhantomJS 2.1 (ariya/phantomjs@3d9a327). Though the binaries need to be built and uploaded.

What platform / OS do you use? If you can build PhantomJS locally as documented in the build instruction, that should do it (and eventually upload the binary).

**Comment by eljefe6a** [comment]
I'm on MacOSX. I followed the instructions to build Phantom (version is `2.2.0-development` btw).

With the new version, I tried to use my original flex CSS `flex`. It didn't work. After googling and trying some things, I finally went through qtwebkit's code. It looks like even at QT 5.5 you still have to prefix everything like `-webkit-flex`. Putting this out there on the interwebs because my google-fu sure didn't find it.
